drivers/src/hand_off.rs

```diff
@@ -247,6 +248,9 @@ pub struct FirmwareHandoffTable {
     /// Index of RT SVN value in the Data Vault
     pub rt_svn_dv_hdl: HandOffDataHandle,
 
+    /// Index of RT Min SVN value in the Data Vault
+    pub rt_min_svn_dv_hdl: HandOffDataHandle,
```

Comment:
please keep documentation in sync by adding field to fmc/README.md

Reply:
Done.